consensus/applytransaction.go

```diff
@@ -13,13 +13,12 @@ func (s *State) applySiacoinInputs(bn *blockNode, t Transaction) {
 			}
 		}
 
-		scod := SiacoinOutputDiff{
-			New:           false,
+		bn.siacoinOutputDiffs = append(bn.siacoinOutputDiffs, SiacoinOutputDiff{
```

Comment:
idk about this man, declaring and then appending seems like a good way to do it.